contracts/satoshi-bridge/src/deposit_msg.rs

```diff
@@ -65,7 +66,7 @@ impl Contract {
         }
         let mut total_gas = 0;
         let mut total_amount = 0;
-        for (index, post_action) in post_actions.iter().enumerate() {
+        for (index, post_action) in post_actions.iter_mut().enumerate() {
```

Comment:
```suggestion
        for (index, post_action) in post_actions.iter_mut().enumerate() {
            post_action.msg = post_action.msg.replace("{{UTXO_TX_ID}}", utxo_storage_key);
```

I think the changes can be done with less code.

Reply:
In general, we can do it this way, it’s really the simplest option. In that case, the replacement will always happen regardless of what is specified in the templates.

I also thought about adding a special template like 'add UTXO', and if such a template exists, then perform the replacement.

Reply:
I think about something like that:
https://github.com/Near-One/btc-bridge/pull/4/commits/3145bfad8216322241a9a9a4d6054611103bd1be

I’d like to isolate the contracts to which the changes are applied.

In this option, the code is simpler. However, the logic becomes stranger. Templates stop being templates and turn into modifiers.

The first option sounds more logical in the context of templates.

Reply:
I also thought that it is important to add additional restrictions, but after thinking more on it, I think we can make things simple and just do replace.
There is a low chance to get collision with this naming.